CHANGELOG.md

```diff
@@ -2,15 +2,65 @@
 
 ## Unreleased
 
-## 💡 Enhancements 💡
+## v0.28.0
+
+# 🎉 OpenTelemetry Collector Contrib v0.28.0 (Beta) 🎉
+
+The OpenTelemetry Collector Contrib contains everything in the [opentelemetry-collector release](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.28.0) (be sure to check the release notes here as well!). Check out the [Getting Started Guide](https://opentelemetry.io/docs/collector/getting-started/) for deployment and configuration information.
+
+## 🚀 New components 🚀
+
+- `humio` exporter to export data to Humio using JSON over the HTTP [Ingest API](https://docs.humio.com/reference/api/ingest/)
+- `udplog` receiver to receives logs from udp using the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library
+- `tanzuobservability` exporter to send traces to [Tanzu Observability](https://tanzu.vmware.com/observability)
 
-- Enabled Dependabot for Github Actions (#3543)
-=======
 ## 🛑 Breaking changes 🛑
 
 - `f5cloud` exporter (#3509):
   - Renamed the config 'auth' field to 'f5cloud_auth'. This will prevent a config field name collision when [Support for Custom Exporter Authenticators as Extensions](https://github.com/open-telemetry/opentelemetry-collector/pull/3128) is ready to be integrated.
 
+## 💡 Enhancements 💡
+
+- Enabled Dependabot for Github Actions (#3543)
+- Change obsreport helpers for receivers to use the new pattern created in Collector (#3439,#3443,#3449,#3504,#3521,#3548)
+- `datadog` exporter:
+  - Add logging for unknown or unsupported metric types (#3421)
+  - Add collector version tag to internal health metrics (#3394)
+  - Remove sublayer stats calc and mutex (#3531)
+  - Deduplicate hosts for which we send running metrics (#3539)
+  - Add support for summary datatype (#3660)
+  - Add datadog span operation name remapping config option (#3444)
+  - Update error formatting for error spans that are not exceptions (#3701)
+- `nginx` receiver: Update the nginx metrics to more closely align with the conventions (#3420)
+- `elasticsearch` exporter: Init JSON encoding support (#3101)
+- `jmx` receiver:
+  - Allow setting system properties (#3450)
+  - Update tested JMX Metric Gatherer release (#3695)
+- Refactor components for the Client Authentication Extensions (#3507)
+- Remove usage of `tracetranslator` bigendian helpers in `jaegerthrifthttp` exporter and `tailsampling` processor (#3451)
```

Comment:
This is an internal change, not interesting for user, let's remove from the changelog.

Reply:
Thanks, line removed